src/Services/OrderService.php

```diff
@@ -161,6 +167,51 @@ public function placeOrder():LocalizedOrder
         return LocalizedOrder::wrap( $order, $this->sessionStorage->getLang() );
 	}
 
+    /**
+     * Subscribe the customer to the newsletter, if stored in the session
+     */
+	public function subscribeToNewsletter()
+    {
+        /** @var CustomerService $customerService */
+        $customerService = pluginApp(CustomerService::class);
```

Comment:
Aus $customerService sollte auch eine Klassenvariable werden, weil sie hier und in placeOrder() verwendet wird.

Reply:
done